src/MyEventEmitter.js

```diff
@@ -1,12 +1,99 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = [];
```

Comment:
```suggestion
    this.events = {};
```
The keys of an object should be event names

Reply:
```
this.events = {
  someEvent1: [cb, cb],
 someEvent2: [cb]
} 
```
- just to demonstrate the data structure here